source/codegen/metadata/nidmm/functions.py

```diff
@@ -0,0 +1,3129 @@
+# -*- coding: utf-8 -*-
+# This file is generated from NI-DMM API metadata version 19.6.0d7
+functions = {
+    'Abort': {
+        'documentation': {
```

Comment:
Remove the 'documentation' tags, its not necessary to retain this.

Reply:
removed documentation tags